backend/src/ee/controllers/v1/workspaceController.ts

```diff
@@ -567,6 +567,83 @@ export const rollbackWorkspaceSecretSnapshot = async (req: Request, res: Respons
  * @param res
  */
 export const getWorkspaceAuditLogs = async (req: Request, res: Response) => {
+  /* 
+    #swagger.summary = 'Return audit logs'
+    #swagger.description = 'Return audit logs'
+    
+    #swagger.security = [{
+      "apiKeyAuth": [],
+      "bearerAuth": []
+    }]
+
+    #swagger.parameters['workspaceId'] = {
+      "description": "ID of the workspace where to get folders from",
+      "required": true,
+      "type": "string",
+      "in": "path"
+    }
+
+    #swagger.parameters['offset'] = {
+      "description": "Number of versions to skip",
+      "required": false,
+      "type": "string"
+    }
+
+    #swagger.parameters['limit'] = {
+      "description": "Maximum number of versions to return",
+      "required": false,
+      "type": "string"
+    }
+
+   #swagger.parameters['startDate'] = {
+      "description": "Filter logs from the provided date",
```

Comment:
When writing API specs, let's assume that users don't know much. In this case, I would not know what format the date should be. Is it 11/2/23 or is in some other format. Can you pease go through all of the specs and make sure there isn't any ambiguity about what format a data type should be in?

Reply:
This is a very good point actually; it’s the little things in making the API usable